Autopilot/SensorFusion/Src/SensorFusion.cpp

```diff
@@ -522,17 +522,7 @@ IMU_Data_t SF_GetRawIMU()
     
     IMU_Data_t imuOutput;
 
-    imuOutput.accx = imuData.accx;
-    imuOutput.accy = imuData.accy;
-    imuOutput.accz = imuData.accz;
-    imuOutput.gyrx = imuData.gyrx;
-    imuOutput.gyry = imuData.gyry;
-    imuOutput.gyrz = imuData.gyrz;
-    imuOutput.magx = imuData.magx;
-    imuOutput.magy = imuData.magy;
-    imuOutput.magz = imuData.magz;
-    imuOutput.isDataNew = imuData.isDataNew;
-    imuOutput.sensorStatus = imuData.sensorStatus;
+    std:memcpy(&imuOutput, &imuData, sizeof(imuOutput));
```

Comment:
I'd prefer if you used `sizeof(STRUCT_YOU_ARE_COPYING)` here instead.

ex. `sizeof(IMU_Data_t)`